README.md

```diff
@@ -23,7 +27,7 @@ Create a section in your `README.md`:
 
 Make sure to have some utility exports in `src/index.ts` annotated with JSDocs.
 
-Now invoke `automd`:
```

Comment:
I think some sort of punctuation here would make sense?

Reply:
This was from the original docs, if you have suggestions please make suggestions 🙏🏼

Reply:
Huh? This is a change you've made in an earlier, non-related part of the docs. You did make the same change lower down though where you've added documentation for this addition though if that's what you were thinking of.

Reply:
The README has changed now with the merge so it doesn't really matter ahah.